hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1724,17 +1724,17 @@
       name: chip_sw_kmac_idle
       desc: '''Verify the KMAC idle signaling to clkmgr.
 
-            - Write the KMAC clk hint to 1 within clkmgr to indicate KMAC clk is ready to be gated.
-              Verify that the KMAC clk hint status within clkmgr reads 0 (KMAC is idle).
+            - Enable the KMAC clk hint within clkmgr to indicate KMAC clk is ready to be gated.
+              Verify that the KMAC clk enabled status within clkmgr reads disabled (KMAC is idle).
             - Initiate an KMAC operation with a known key, plain text and digest.
-              Verify that the KMAC clk hint status within clkmgr now reads 1 (KMAC is not idle),
-              before the KMAC operation is complete.
+              Verify that when enabling the KMAC clk hint after the operation has started but is not
+              yet complete that the clk enabled status in clkmgr stays enabled (KMAC is not idle).
             - After the KMAC operation is complete, verify the digest for correctness.
-              Verify that the KMAC clk hint status within clkmgr now reads 0 again (KMAC is idle),
-              after the KMAC operation is complete.
+              Verify that the KMAC clk enabled status reads as disabled when enabling
+              the hint again (KMAC is idle).
```

Comment:
```suggestion
              Verify that the KMAC clk enabled status reads as disabled (KMAC is idle).
```

This is because you had enabled the hint already, and had not modified it.